### PR TITLE
chore(aqua): replace the local Registry to the Stanard Registry

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**
- https://github.com/morning-night-guild/platform-app/issues/83

### CODE REVIEW

File: aqua.yaml [hunk 1]
```diff
@@ -3,24 +3,13 @@
...
-  - name: google.golang.org/protobuf/cmd/protoc-gen-go
-    registry: local
-    version: v1.28.1
-  - name: github.com/bufbuild/connect-go/cmd/protoc-gen-connect-go
-    registry: local
-    version: v1.1.0
-  - name: github.com/k1LoW/tbls
-    registry: local
-    version: v1.62.0
+  - name: deepmap/oapi-codegen@v1.12.4
```
Comment (suzuki-shunsuke) [inline aqua.yaml:12]: https://github.com/aquaproj/aqua-registry/blob/main/pkgs/deepmap/oapi-codegen/registry.yaml

File: aqua.yaml [hunk 2]
```diff
@@ -3,24 +3,13 @@
...
+  - name: protocolbuffers/protobuf-go/protoc-gen-go@v1.28.1
+  - name: bufbuild/connect-go/protoc-gen-connect-go@v1.1.0
```
Comment (suzuki-shunsuke) [inline aqua.yaml:14]: https://github.com/aquaproj/aqua-registry/blob/main/pkgs/protocolbuffers/protobuf-go/protoc-gen-go/registry.yaml

File: aqua.yaml [hunk 3]
```diff
@@ -3,24 +3,13 @@
...
-    registry: local
-    version: v1.28.1
-  - name: github.com/bufbuild/connect-go/cmd/protoc-gen-connect-go
-    registry: local
-    version: v1.1.0
-  - name: github.com/k1LoW/tbls
-    registry: local
-    version: v1.62.0
+  - name: deepmap/oapi-codegen@v1.12.4
+  - name: protocolbuffers/protobuf-go/protoc-gen-go@v1.28.1
```
Comment (suzuki-shunsuke) [inline aqua.yaml:13]: https://github.com/aquaproj/aqua-registry/blob/main/pkgs/protocolbuffers/protobuf-go/protoc-gen-go/registry.yaml

**Comment by takokun778** [comment]
@suzuki-shunsuke 

Thank you for not only providing the information, but also for your support in replacing it.
The Aqua project is very easy to use and I like it very much. Please keep up the good work.

**Pull request merged**